src/routes/computePlanDetails/components/CancelComputePlanMenuItem.tsx

```diff
@@ -2,15 +2,28 @@ import { MenuItemProps, MenuItem, Tooltip } from '@chakra-ui/react';
 
 type CancelComputePlanMenuItemProps = {
     onClick: MenuItemProps['onClick'];
-    isDisabled: boolean;
+    hasPermissions: boolean;
+    hasCancellableStatus: boolean;
 };
 const CancelComputePlanMenuItem = ({
     onClick,
-    isDisabled,
-}: CancelComputePlanMenuItemProps) =>
-    isDisabled ? (
+    hasPermissions,
+    hasCancellableStatus,
+}: CancelComputePlanMenuItemProps) => {
+    const isDisabled = !hasPermissions || !hasCancellableStatus;
```

Comment:
Nitpick: it might be cleaner to replace it with `isEnabled = hasPermissions && hasCancellableStatus`